Package.swift

```diff
@@ -7,7 +7,9 @@ let package = Package(
     name: "sbg",
     dependencies: [
         .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMinor(from: "0.13.0")),
-        .package(url: "https://github.com/tuist/xcodeproj.git", .upToNextMajor(from: "6.0.0")),
+//        .package(url: "https://github.com/tuist/xcodeproj.git", .upToNextMajor(from: "6.0.0")),
```

Comment:
Seems like you need to cover your changes with unit test and then they will merge your PR :P

Reply:
Yep, I did it today